redash/query_runner/azure_kusto.py

```diff
@@ -0,0 +1,159 @@
+from redash.query_runner import BaseQueryRunner, register
+from redash.query_runner import TYPE_STRING, TYPE_DATE, TYPE_DATETIME, TYPE_INTEGER, TYPE_FLOAT, TYPE_BOOLEAN
+from redash.utils import json_dumps, json_loads
+
+
+try:
+    from azure.kusto.data.request import KustoClient, KustoConnectionStringBuilder
+    from azure.kusto.data.exceptions import KustoServiceError
+    enabled = True
+except ImportError:
+    enabled = False
+
+TYPES_MAP = {
+    'boolean': TYPE_BOOLEAN,
+    'datetime': TYPE_DATETIME,
+    'date': TYPE_DATE,
+    'dynamic': TYPE_STRING,
+    'guid': TYPE_STRING,
+    'int': TYPE_INTEGER,
+    'long': TYPE_INTEGER,
+    'real': TYPE_FLOAT,
+    'string': TYPE_STRING,
+    'timespan': TYPE_STRING,
+    'decimal': TYPE_FLOAT
+}
+
+
+class AzureKusto(BaseQueryRunner):
+    noop_query = "let noop = datatable (Noop:string)[1]; noop"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type":
+            "object",
+            "properties": {
+                "Cluster": {
+                    "type": "string"
+                },
+                "Azure AD Client ID": {
```

Comment:
We use `snake_case` (`azure_ad_client_id`) for properties name and then provide a friendly name as the `title` property.

Reply:
Fixed in https://github.com/getredash/redash/pull/4091/commits/b969c501923495f769c6eb3bcd68b7177e8ab5dc